driver/driver.go

```diff
@@ -89,3 +92,26 @@ func Boot(ctx, clientContext context.Context, d Driver, pw progress.Writer) (*cl
 		return c, nil
 	}
 }
+
+func HistoryAPISupported(ctx context.Context, c *client.Client) (res bool) {
+	res = true
+	checkErrF := func(err error) {
+		if s, ok := grpcerrors.AsGRPCStatus(err); ok {
+			if s.Code() == codes.Unimplemented {
+				res = false
+			}
+		}
+	}
+	cl, err := c.ControlClient().ListenBuildHistory(ctx, &controlapi.BuildHistoryRequest{
+		ActiveOnly: true,
+		Ref:        "buildx-dummy-ref", // dummy ref to check if the server supports the API
```

Comment:
"buildx-test-history-api-feature" so that if you look at it from server side you understand where the request is coming from.